.github/workflows/format-markdown-code.yml

```diff
@@ -0,0 +1,57 @@
+name: 'Format Markdown code'
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  format:
+    name: 'Format Markdown code'
+    runs-on: ubuntu-latest
+    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/format')
+    steps:
+      - name: 'Checkout code'
+        run: |
+          PR_DATA="/tmp/pr.json"
+
+          jq -r ".issue.pull_request.url" "$GITHUB_EVENT_PATH" | \
+            xargs curl --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' -o "$PR_DATA" --url
+
+          HEAD_REF=$(jq -r ".head.ref" "$PR_DATA")
+          HEAD_REPO=$(jq -r '.head.repo.clone_url | sub("https://"; "https://x-access-token:${{ secrets.GITHUB_TOKEN }}@")' "$PR_DATA")
+
+          git clone $HEAD_REPO .
+          git checkout -b "$HEAD_REF" "origin/$HEAD_REF"
+
+      - name: 'Format Markdown code'
+        run: |
+          # Use prettier to format the added or modified Markdown files in the PR (if any)
+          jq -r '.issue.pull_request.url | . += "/files"' "$GITHUB_EVENT_PATH" | \
+            xargs curl --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' --url | \
+            jq -c '.[] | select(.status == "added" or .status == "modified") | select(.filename | match("\\.md$")) | .filename' | \
+            xargs npx prettier@2.0.4 --write
+
+      - name: 'Commit formatted Markdown code'
+        run: |
+          # Check if there is nothing to commit (i.e. no formatting changes made)
+          if [ -z "$(git status --porcelain)" ]; then
+            echo "Code is already formatted correctly"
+            exit 0
+          fi
+
+          # Setup the git user (required to commit anything)
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git config --global user.name "github-actions[bot]"
+
+          # Commit the changes made by prettier
+          git add .
+          git commit -m "[CI] Format Markdown code"
+          git push
+
+  # https://github.community/t5/GitHub-Actions/Workflow-is-failing-if-no-job-can-be-ran-due-to-condition/m-p/38186#M3250
```

Comment:
The diffs got a bit messed up due to the rename. I think this job can be removed again since the original version in this PR didn't have it and still worked?

Reply:
No I didn't check if it runs without any matches, and the corresponding GH issue hasn't been updated yet.

Reply:
I checked it in your fork and it just showed that all checks were skipped, but no failure

Reply:
It did? Oh nice! I've removed it.